prometheus_client/exposition.py

```diff
@@ -10,18 +10,19 @@
 from wsgiref.simple_server import make_server
 
 from . import core
+
+from .handlers.base import handler as default_handler
```

Comment:
Not sure I understand. Do you mean something like...

```
from .handlers.base import default_handler
```

and in base.py...

```
default_handler = handler
```

or something else???